test/run_test.py

```diff
@@ -134,6 +134,8 @@ def skip_test_p(name: str) -> bool:
         "distributed/elastic/utils/distributed_test",
         "distributed/elastic/multiprocessing/api_test",
         "test_deploy",
+        "lazy/test_misc.py",
```

Comment:
@Krovatkin you shouldn't have to! These should be picked up already by default as our normal way of garnering tests is sweeping the test directory for files starting with `test_`

Reply:
this sounds awesome!

Reply:
To verify, you can grep for `run_test.py` in the logs of any of the default CI jobs and check that test_misc.py is getting run in the selected tests list for one of the shards.

Reply:
@janeyx99 thanks! I expect this test to fail for now until another PR with the fix is checked in :) and it's failing alright : https://github.com/pytorch/pytorch/runs/6199967065?check_suite_focus=true